src/accountingservice/go.mod

```diff
@@ -5,12 +5,12 @@ go 1.19
 require (
 	github.com/Shopify/sarama v1.37.2
 	github.com/sirupsen/logrus v1.9.0
-	go.opentelemetry.io/contrib/instrumentation/github.com/Shopify/sarama/otelsarama v0.36.3
-	go.opentelemetry.io/otel v1.11.0
+	go.opentelemetry.io/contrib/instrumentation/github.com/Shopify/sarama/otelsarama v0.36.1
```

Comment:
why otelsarama package is downgrade?

Reply:
When I ran the command `go get -u go.opentelemetry.io/otel/`, Go automatically downgraded the otelsarama package.